pylsp/text_edit.py

```diff
@@ -0,0 +1,90 @@
+def get_well_formatted_range(lsp_range):
```

Comment:
```suggestion
# Copyright 2017-2020 Palantir Technologies, Inc.
# Copyright 2021- Python Language Server Contributors.

def get_well_formatted_range(lsp_range):
```

Missing header

Reply:
Will fix it up.

However, you should consider removing the copyright headers, unless we're distributing files individually, they add noise to MIT projects for no return. The License.md file should be good enough unless a specific file has different licensing from the root license.

Reply:
You're probably right, but I think it's easier to add this header for new files than remove them from all of them.